glue_solar/core/sunpy_maps/sunpy_maps_loader.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
Would calling it `loader.py` be enough?

Reply:
I have a funny feeling we should keep the different loaders separate, since otherwise they would be harder to maintain in the near future.

Reply:
I am not saying we merge them but the folder names give away what loader this is for without having to replicate the folder name into the filename.

Reply:
Oh I see, let me fix them...

Reply:
Yup, done